tests/tests_basics_forwards.yml

```diff
@@ -33,6 +30,25 @@
     __test_template_sys: RSYSLOG_SyslogProtocol23Format
 
   tasks:
+    - name: create tempdir for test files
+      tempfile:
+        state: directory
+        prefix: logging_test_
+      delegate_to: localhost
+      register: _tmpdir
```

Comment:
where is `__tmpdir.path` removed?  The usual way this is done is like this:
```
- name: a block of tasks that use a temp file
  block:
    - name: create tempdir
      tempfile:
        ...
      register: __tmpdir

    - name: use the temp file
    ...
  always:
    - name: remove tempdir
      file:
        path: "{{ __tmpdir.path }}"
        state: absent
```
That way the tmpdir is removed even if the test fails

Reply:
Thanks, @richm. Fixed.
(I wish the tempdirs were automatically removed at the end of the ansible execution... :p)